packages/grid/_modules_/grid/models/params/gridRowScrollEndParams.ts

```diff
@@ -0,0 +1,23 @@
+import { GridColumns } from '../colDef/gridColDef';
+
+/**
+ * Object passed as parameter in the onRowsScrollEnd callback.
+ */
+export interface GridRowScrollEndParams {
+  /**
+   * The number of rows that fit in the viewport.
+   */
+  viewportPageSize: number;
+  /**
+   * The number of rows allocated for the rendered zone.
+   */
+  virtualRowsCount: number;
+  /**
+   * The grid visible columns.
+   */
+  visibleColumns: GridColumns;
+  /**
+   * API ref that let you manipulate the grid.
+   */
+  api: any;
```

Comment:
As I side note. I don't understand why we expose the api ref in the params. There is already a dedicated `apiRef` prop. Shouldn't this be enough? It's also not supposed to be available in the MIT DataGrid.

Reply:
x2 Everything that I added in the params can be retrieved from the gird API some way or another. Besides the event itself, I don't find it necessary for those to be exposed like this.

On another note - all of the existing callbacks return the API.

Reply:
> Everything that I added in the params can be retrieved from the gird API some way or another

Well, I guess it depends, how easily can developers retrieve these other params? And how relevant are they for the given callback? But I do agree that providing multiple API, **if** solving the same problem harms.

Reply:
It isn't difficult. My original demo wasn't using anything from the props to be functional (granted that it was a very simplistic view but it showcases that it is not difficult to achieve that without any additional params needed).

Reply:
I'm resolving the discussion as it's not directly related to the changes. I have saved the topic. We can talk about it during the next meeting.